File: Application/Services/Operations/Finances/Dtos/Bank/CreditCardLimitOperationDto.cs
using System;
using Application.Services.Operations.Authentication.Dtos;
using Application.Services.Operations.Main.Companies.Dtos;
using Application.Services.Shared.Dtos;

namespace Application.Services.Operations.Finances.Dtos.Bank
{
    public class CreditCardLimitOperationDto : RootBaseDto
    {

        public int CardId { get; set; }
        public CardDto Card { get; set; }
        public decimal LimitCreditUsed { get; set; }
        public decimal PriceOfLastPayment { get; set; }
        public DateTime LastPayment { get; set; }
    }
}

File: Views.old/src/components/financial/components/bank-account-cards/dto/credit-card-limit-operation-dto.ts
import { CompanyDto } from "src/shared/entities-dtos/company-dto";
import { CardDto } from "./card-dto";
import { MyUser } from "src/components/authentication/dto/my-user";
import { RootBase } from "src/shared/entities-dtos/root-base";

export class CreditCardLimitOperationDto extends RootBase {
    cardId: number;
    card: CardDto;
    limitCreditUsed: number;
    priceOfLastPayment: number;
    lastPayment: Date;
}
